bin/convert-argv.js

```diff
@@ -553,5 +554,11 @@ module.exports = function(optimist, argv, convertOptions) {
 			console.error("Use --help to display the CLI options.");
 			process.exit(-1); // eslint-disable-line
 		}
+		var webpackOptionsValidationErrors = validateWebpackOptions(options);
+
+		if (webpackOptionsValidationErrors.length) {
+			console.error("Configuration is invalid. Fix the following errors:", webpackOptionsValidationErrors);
+			process.exit(-1); // eslint-disable-line
+		}
```

Comment:
Have you seen the comment about this?

https://github.com/webpack/webpack/pull/2974/commits/efc4113e12b3e1677f4f1b4bc54899b8f3abb8fb

Please advise if to proceed in your direction after you have read it.